documentation/notifications.md

```diff
@@ -18,13 +18,53 @@ Notifications are set up in your credentials JSON file. They will use the `notif
         ...
   } ,
   "notifications": {
-      "slack_url": "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX",
+      "slack_url": "https://api.slack.com/apps/2RFP2A7EN1/incoming-webhooks",
       "teams_url": "https://outlook.office.com/webhook/00000000-0000-0000-0000-000000000000@00000000-0000-0000-0000-000000000000/IncomingWebhook/00000000000000000000000000000000/00000000-0000-0000-0000-000000000000"
   }
 ```
 {% endcode %}
 
-You also must run `dnscontrol preview` or `dnscontrol push` with the `-notify` flag to enable notification sending at all.
+## Usage
+
+If you want to send a notification, add the `--notify` flag to the `dnscontrol preview` or `dnscontrol push` commands.
+
+Below is an example where we add [the A record](functions/domain/A.md) `notify` and display the notification output.
+
+{% code title="dnsconfig.js" %}
+```diff
+D("example.com", REG_MY_PROVIDER, DnsProvider(DSP_MY_PROVIDER),
++ A('notify', '1.2.3.4'),
```

Comment:
New users might think that the A() record is somehow required for notify to work. Suggestion: use a DNS label such as `"example"`

Reply:
I can imagine that, I have changed the example from A-record `notify` to `foo` (_similar to the example on [domain modifier A](https://docs.dnscontrol.org/language-reference/domain-modifiers/a)_).